tests/integration_tests/test_integration.py

```diff
@@ -141,16 +176,28 @@ def check_base_install():
     subprocess.CalledProcessError
         If the installation check script returns a non-zero exit status.
     """
-    base_install_check = [
+    cmd = [
         "bash",
         os.path.join(
             os.getenv("GEOIPS_PACKAGES_DIR"),
             "geoips/tests/integration_tests/base_install.sh",
         ),
         "exit_on_missing",
+        "skip_create_registries",
```

Comment:
I skip creating registries from pytest, it can take a bit when there many plugin repos